README.md

```diff
@@ -53,5 +53,90 @@ with ease.
 ---
 
 ### Current Status
+ 
+We are preparing to start publishing RPM packages publicly soon. It's possible to build included packages manually.
+
+
+### Development
+
+To build the packages, use the [exe/pgpm](exe/pgpm) script.
+
+Example:
+
+```sh
+./exe/pgpm build pgvector
```

Comment:
Just a minor note, the base image doesn't need this `exe` thing – the gem is installed there. This format is only for the dev image.